src/pb_view.cpp

```diff
@@ -75,20 +77,14 @@ void pb_view::run(bool auto_download)
 				ctrl->downloads().size());
 
 			std::string code = "{list";
+			std::string formatstring = ctrl->get_formatstr();
+
+			unsigned int width = utils::to_u(dllist_form.get("feeds:w"));
 
 			unsigned int i = 0;
 			for (const auto& dl : ctrl->downloads()) {
 				auto lbuf = strprintf::fmt(
```

Comment:
This `strprintf::fmt` can be removed now, since `format_line` already returns a string and we don't need to format it further.